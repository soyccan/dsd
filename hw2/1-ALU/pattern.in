695c7
24cad
a4b2d
71187
c1ef9
bb8bf
38706
79408
2c187
8fda2
5e7c5
39cb5
25313
a5ad2
00685
3d3f1
6ab43
78268
4fdb1
6ccd5
0e734
461da
0ac29
24051
7c01b
98c51
65c75
77d43
87d34
abcca
1a4cd
b0737
8c923
53c45
927c5
b3a86
956a7
b243d
391dc
1e140
6829f
abd41
bfd5f
c8692
67755
a55a0
11eb4
055e2
2b87f
bb9f7
