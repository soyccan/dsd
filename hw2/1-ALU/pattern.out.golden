128
00c
096
10e
000
05c
087
080
081
005
022
0bd
013
0b4
18b
0f3
114
1a0
102
122
01b
19e
1d5
040
01b
1c6
182
060
001
179
1d7
183
011
079
013
11d
02b
012
0dd
1a1
160
17b
0fe
000
188
0aa
06a
037
038
0dc
